src/web_app_skeleton/tasks/db/seed/sample_data.cr

```diff
@@ -8,21 +8,22 @@ class Db::Seed::SampleData < LuckyTask::Task
   summary "Add sample database records helpful for development"
 
   def call
-    # Using a Avram::Factory:
+    # Using an Avram::Factory:
     #
     # Use the defaults, but override just the email
     # UserFactory.create &.email("me@example.com")
 
     # Using a SaveOperation:
-    #
-    # SaveUser.create!(email: "me@example.com", name: "Jane")
+    # ```
+    # SignUpUser.create!(email: "me@example.com", password: "test123", password_confirmation: "test123")
+    # ```
     #
     # You likely want to be able to run this file more than once. To do that,
     # only create the record if it doesn't exist yet:
-    #
-    # unless UserQuery.new.email("me@example.com").first?
-    #  SaveUser.create!(email: "me@example.com", name: "Jane")
-    # end
+    # ```
+    # # be sure to set `upsert_lookup_columns :email` in the operation
+    # SignUpUser.upsert!(email: "me@example.com", password: "test123", password_confirmation: "test123")
```

Comment:
This doesn't do the same thing that the comment says. Upsert will update, but the comment is referring to a find or create.

Reply:
I was trying to be clever 😛  I guess I could change the example to do `if UserQuery.new.email().none?` and then use the normal Signupuser.create?

Reply:
Yea, that's what I was thinking